src/panelMenuButton.js

```diff
@@ -55,6 +56,8 @@ var PanelMenuButton = GObject.registerClass(
         }
 
         _initTimers() {
+            let prefs = (new Settings.Prefs());
```

Comment:
Let us init settings in constructor: 
```js
...
this.cpu = new Cpu();
this.iconProvider = new IconProvider();
this.settings = new Settings(); // Added
this.sleepingThreshold = this.settings.sleepingThreshold.get();  // Added
...
```
Here we add listener instead of `let prefs = (new Settings.Prefs());`:
```js
this.settings.sleepingThreshold.addListener(() => {
       this.sleepingThreshold = this.settings.sleepingThreshold.get();
});
```

And update condition below:
```js
this.cpu.utilization > this.sleepingThreshold ? this.iconProvider.nextSprite : this.iconProvider.sleeping,
```